### PR TITLE
Updated LoadElementalAnalysisData algorithm 

### LINKED ISSUE #31807: Update LoadElementalAnalysisData algorithm
LoadElementalAnalysisData algorithm should have the capability to add errors automatically to loaded workspaces

### PR DESCRIPTION
**Pull request by tolu28-coder**
**Description of work.**
LoadElementalAnalysisData Algorithm was updated to add errors to workspaces loaded

**To test:**
1.  Download the test files from PR #29958 and add to User Directories.
2.  Load data using LoadElementalAnalysisData algorithm .
3.  Show data in workspace and errors should be the square root of the y data.

Fixes #31807. 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

**Review by sf1919 — CHANGES_REQUESTED** [review]
I ran through the tests and everything seems to work. Also tested it on some other Elemental Analysis Data I have as the test files have zero values for Prompt and Delay on each Detector and I wanted to be sure the E values were calculated for everything. Good news - it all looks good.

Only one set of changes I'd like to see. In the tests you'll see there is a note explaining what the assertion line is checking for e.g. ```# Check that the Bin 4083 of the Total spectra of the workspace of Detecror 4 is 120.457``` . Please can you add relevant notes to the new checks for errors in the tests?

File: docs/source/release/v6.2.0/muon.rst [hunk 1]
```diff
@@ -13,4 +13,11 @@ Improvements
 
 - It is now possible to do a vertical resize of the plot in Muon Analysis and Frequency Domain Analysis.
 
+Elemental Analysis
+------------------
+
+Improvements
+############
+- Updated :ref:`LoadElementalAnalysisData <algm-LoadElementalAnalysisData>` algorithm to add errors to workspace.
```
Comment (AnthonyLim23) [inline docs/source/release/v6.2.0/muon.rst:21]:
```suggestion
- Updated :ref:`LoadElementalAnalysisData <algm-LoadElementalAnalysisData>` algorithm to include Poisson errors for the counts data.
```

**Review by sf1919 — APPROVED** [review]

**Comment by sf1919** [comment]
Thanks for making the requested changes.

The code looks good, with relevant details in tests and release documentation. It's a much needed addition to the LoadElementalAnalysis algorithm.
